perception/tensorrt_yolox/launch/yolox_s_plus_opt.launch.xml

```diff
@@ -1,8 +1,9 @@
 <?xml version="1.0"?>
 <launch>
+  <!-- cspell:ignore finetune -->
   <arg name="input/image" default="/sensing/camera/camera0/image_rect_color"/>
   <arg name="output/objects" default="/perception/object_recognition/detection/rois0"/>
-  <arg name="model_name" default="yolox-sPlus-opt"/>
+  <arg name="model_name" default="yolox-sPlus-T4-960x960-pseudo-finetune.onnx"/>
```

Comment:
```suggestion
  <arg name="model_name" default="yolox-sPlus-T4-960x960-pseudo-finetune"/>
```

@badai-nguyen 
Thank you for adding the updated model.

I believe file extensions (i.e., `.onnx`) should not be included in the value of `model_name` since the extension is added [here](https://github.com/autowarefoundation/autoware.universe/blob/20d1598e07f05f8456195ce56a0ffe9768246f7c/perception/tensorrt_yolox/launch/yolox_s_plus_opt.launch.xml#L44).
Could you please check here again?

Reply:
@manato thank you for your review. It was a typo. I fixed at https://github.com/autowarefoundation/autoware.universe/pull/4949/commits/a43f06b9149a24d82073ce8010b68f7bf3782343